src/js/templates/maps/expansion-panel.html

```diff
@@ -1,6 +1,8 @@
 <div class="<%= classNames.toggle %>">
   <div class="<%= classNames.icon %>">
-    <i class="icon <%= icon %>"></i>
+    <%if(!isSvgIcon) {%>
+      <i class="icon <%= icon %>"></i>
+    <%}%>
```

Comment:
does it work if we just do

```
<% if (isSvgIcon) { %>
  <%= icon %>
<% } else { %>
  <i ...></i>
<% } %>
```

Reply:
Combined this with default SVG icon

Reply:
ah the default is a layer icon that might only make sense for layer categories. I'm not sure if we want it to apply to all expansion panel. might be better if we pass it in from layer category list?

Reply:
Solved by resolving a TODO in AssetCategory.js setting default icon